src/modules/gift/dto/obligation.ts

```diff
@@ -50,10 +50,10 @@ export class GiftObligationDto {
   @Min(VALIDATION.OBLIGATION_AMOUNT.MIN)
   @Max(VALIDATION.OBLIGATION_AMOUNT.MAX)
   @ApiProperty({
-    example: OBLIGATION().obligationAmount,
+    example: OBLIGATION().amount,
     required: true,
   })
-  obligationAmount: number;
+  amount: number;
```

Comment:
Are we keeping this aligned with the rest of the endpoints where all the amount fields will be called `amount`?